titan/model.py

```diff
@@ -129,7 +132,7 @@ def print_stats(self, stat: Dict[str, Dict[str, int]], outdir: str):
                 )
 
     def reset_trackers(self):
-        self.deaths = []
+        self.exits = dict((k, []) for k in self.exits)
```

Comment:
```suggestion
        self.exits = { exit: [] for exit in self.exits }
```
same result, but keeps a more consistent code style